apps/dav/lib/CalDAV/Schedule/Plugin.php

```diff
@@ -272,12 +288,13 @@ function propFindDefaultCalendarUrl(PropFind $propFind, INode $node) {
 				}
 
 				if (strpos($principalUrl, 'principals/users') === 0) {
-					$uri = CalDavBackend::PERSONAL_CALENDAR_URI;
-					$displayname = CalDavBackend::PERSONAL_CALENDAR_NAME;
+					list(, $UID) = split($principalUrl);
```

Comment:
No need to scream :scream: 

```suggestion
					list(, $userId) = split($principalUrl);
```